recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ source:
 build:
     number: {{ build }}
     script: python setup.py install --single-version-externally-managed --record record.txt
+    noarch: python
     entry_points:
         - pystache=pystache.commands.render:main
         - pystache-test=pystache.commands.test:main
```

Comment:
Please add spaces around each `=` for the `entry_points`. Should fix the build issue.